### PR TITLE
Upgrade and edit functionality

### PR DESCRIPTION
**Pull request by moritzkirstein**
Fixes #

## Proposed Changes

  - upgrade nautilus
  - add edit example
  - add "genx" network config (duplicate "pontusx" config)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,10 +18,14 @@ A TypeScript project showing [Nautilus](https://github.com/deltaDAO/nautilus) us
 
    | Variable    | Value                                                           |
    | ----------- | --------------------------------------------------------------- |
-   | NETWORK     | `mumbai` or `pontusx`                                           |
+   | NETWORK     | `MUMBAI`, `GENX`  or `PONTUSX`                                           |
    | PRIVATE_KEY | private key of your account (you can export this from MetaMask) |
 
-   > The account needs to be funded with tokens. Depending on the example and network either `matic` (`mumbai`) or `gx` (`pontusx`) for the network fees and `EUROe` or `OCEAN` for paid services.
+   > The account needs to be funded with tokens. Depending on the example and network either `matic` (`mumbai`) or `gx` (`genx`, `pontusx`) for the network fees and `EUROe` or `OCEAN` for paid services.
```
Comment (Abrom8) [inline README.md:24]: blocking: Pontus-X does not use `gx` fee tokens.

File: config.ts [hunk 2]
```diff
@@ -83,6 +98,35 @@ export const PRICING_CONFIGS: PricingConfig = {
...
+        fixedRateAddress: '0x8372715D834d286c9aECE1AcD51Da5755B32D505',
+        baseTokenAddress: '0x8A4826071983655805bF4f29828577Cd6b1aC0cB',
+        baseTokenDecimals: 18, // adapted for EUROe decimals
+        datatokenDecimals: 18,
+        fixedRate: '1',
+        marketFee: '0',
+        marketFeeCollector: '0x0000000000000000000000000000000000000000'
+      }
+    }
+  },
```
Comment (Abrom8) [inline config.ts:129]: question: Are these the correct addresses?

File: index.ts [hunk 3]
```diff
@@ -44,12 +45,21 @@ async function main() {
...
   // PUBLISH FUNCTIONS
-  await publishAccessDataset(nautilus, networkConfig, pricingConfig, wallet)
-  await publishComputeDataset(nautilus, networkConfig, pricingConfig, wallet)
-  await publishComputeAlgorithm(nautilus, networkConfig, pricingConfig, wallet)
+  // await publishAccessDataset(nautilus, networkConfig, pricingConfig, wallet)
+  // await publishComputeDataset(nautilus, networkConfig, pricingConfig, wallet)
+  // await publishComputeAlgorithm(nautilus, networkConfig, pricingConfig, wallet)
+
+  // EDIT SERVICE OFFERING
+  await editServiceOffering(nautilus)
```
Comment (Abrom8) [inline index.ts:53]: nitpick: Comment out, maybe comment simple publish in.

**Review by Abrom8 — CHANGES_REQUESTED** [review]

File: edit.ts [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    //     })
+
+
+    // Updating trusted algorithms on a service
+    // const serviceBuilder = new ServiceBuilder({ aquariusAsset, serviceId: aquariusAsset.services[0].id })
+    // const service = serviceBuilder
+    //.addTrustedAlgorithmPublisher('0x28080F654eED6CC00e8b16F4841E92CD0c2C0778')
+    //    .build()
+    // const asset = assetBuilder.addService(service)
+    //     .build()
```
[the remark this answers is not in the document]
Comment (moritzkirstein) [reply edit.ts]: refactored to provide more examples

**Review by Abrom8 — APPROVED** [review]

**Pull request merged**